tomcat9/src/test/java/com/linecorp/armeria/server/tomcat/TomcatServiceDestroyedConnectorTest.java

```diff
@@ -46,16 +47,21 @@ protected void configure(ServerBuilder sb) throws Exception {
             tomcatWithWebApp.start();
             sb.serviceUnder("/api/", TomcatService.of(tomcatWithWebApp.getConnector()));
         }
+
+        @Override
+        protected boolean runForEachTest() {
+            return true;
+        }
     };
 
-    @Test
+    @RepeatedTest(100)
```

Comment:
Revert 😄

Reply:
Ah, I'm testing to check if it happens again. 😄